zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/SchemaResourceSerializer.java

```diff
@@ -18,11 +18,14 @@
 import java.util.function.Function;
 
 public final class SchemaResourceSerializer {
-  private static final ObjectMapper MAPPER = new ObjectMapper();
 
-  private SchemaResourceSerializer() {}
+  private final ObjectMapper objectMapper;
 
-  public static Map<String, Object> serialize(
+  public SchemaResourceSerializer(final ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
```

Comment:
❓ Should we also use here the reader?

Reply:
For some reason there is no method in the reader `readValue(String, TypedReference)` :shrug: